version/version.go

```diff
@@ -12,6 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package version
 
-const Version = "0.5.3+git"
+var Version = "0.5.3+was-not-built-properly"
```

Comment:
Kind of unpacking a can of worms I didn't want to, but thoughts on etcd's
approach?
```
etcd Version: 2.2.1+git
Git SHA: Not provided (use ./build instead of go build)
Go Version: go1.5.1
Go OS/Arch: linux/amd64
```

On Wed, Oct 21, 2015 at 6:42 PM, Eugene Yakubovich <notifications@github.com
> wrote:

> In version/version.go
> <https://github.com/coreos/flannel/pull/349#discussion_r42703984>:
>
> >
> > -const Version = "0.5.3+git"
> > +var Version = "0.5.3+was-not-built-properly"
>
> Yeah but then we still need the bump revision dance. I like that git
> describe magic as it cleans it up.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/flannel/pull/349/files#r42703984>.
>